src/Test/Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -88,21 +84,10 @@ public static MetadataReference SystemThreadingTaskFacadeRef
             }
         }
 
-        protected bool PrintActualDiagnosticsOnFailure
-        {
-            set
-            {
-                _printActualDiagnosticsOnFailure = value;
-            }
-        }
+        protected bool PrintActualDiagnosticsOnFailure { private get; set; }
```

Comment:
I thought pattern for things like this is using SetPrintActualDiagnosticsOnFailue(bool). anyway, other seems thinks differently.